src/aspire/image/image.py

```diff
@@ -186,6 +186,40 @@ def __init__(self, data, dtype=None):
         self.__array_interface__ = self._data.__array_interface__
         self.__array__ = self._data
 
+    def project(self, angles):
+        """
+        Computes the Radon Transform on an Image Stack using Non-Uniform Fast Fourier Transforms. This method projects the Image stack along different angles and returns the Radon Transform.
+
+        :param angles: A 1-D Numpy Array of angles in Radians. This is used to compute the Radon Transform at different angles.
+        :return: Radon transform of the Image Stack.
+        :rtype: Ndarray (stack size, number of angles, image resolution)
+        """
+        # number of points to sample on radial line in polar grid
+        n_points = self.resolution
+        original_stack = self.stack_shape
+
+        # 2-D grid
+        y_idx = np.fft.rfftfreq(n_points) * np.pi * 2
+        n_real_points = len(y_idx)
+
+        pts = np.empty((2, len(angles), n_real_points), dtype=self.dtype)
```

Comment:
That's a great point. I think I'll assign it as `n_angles = len(angles)` since `n_points` could be confusing to someone adding to the codebase given there's`n_real_points` as well.

Reply:
Oops, yeah, I think I meant `n_angles`! Good call!